packages/notarization/manager.go

```diff
@@ -48,6 +48,9 @@ func NewManager(epochManager *EpochManager, epochCommitmentFactory *EpochCommitm
 		pendingConflictsCount:  make(map[epoch.EI]uint64),
 		log:                    options.Log,
 		options:                options,
+		Events: &Events{
+			ECCommitable: event.New[*ECCommitableEvent](),
```

Comment:
we should do Epoch commitmed event that is triggered when epoch commitment is first time requested.
And we increase the lastCommitmed epoch index